<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="control">
       <attribute name="icon">
        <iconset theme="applications-system">
         <normaloff>.</normaloff>.</iconset>
       </attribute>
       <attribute name="title">
        <string>Control</string>
       </attribute>
       <widget class="QFrame" name="temperature_chart">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>721</width>
          <height>491</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2"/>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>580</y>
          <width>103</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
       <widget class="QSlider" name="conveyer_units_per_minute_slider">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>40</y>
          <width>160</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>950</x>
          <y>30</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="analytics">
       <attribute name="icon">
        <iconset theme="x-office-presentation">
         <normaloff>.</normaloff>.</iconset>
       </attribute>
       <attribute name="title">
        <string>Analytics</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>400</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>test</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>conveyer_units_per_minute_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>870</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>992</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
